crorrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize
+    name[0..10]
+  end
```

Comment:
- [ ] Your method does not return a capitalized string. Kindly fix that. You can resolve it by adding a bang sign (`!`) to the `capitalize` method.
- [ ] Your method is also supposed to return only 10 character string but it returns 11. Kindly fix this. `[0..9]` or `[0...10]` should work.

  ![elshamy-corr](https://user-images.githubusercontent.com/36542986/156204122-c350b2c6-08c2-47f2-8535-c428cffdb601.jpg)